src/app/signin/page.tsx

```diff
@@ -0,0 +1,112 @@
+'use client';
+import { Button } from '@/components/ui/button';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { Input } from '@/components/ui/input';
+import { Label } from '@/components/ui/label';
+import { signIn } from 'next-auth/react';
+import { useRouter } from 'next/navigation';
+import React, { useRef, useState } from 'react';
+
+const Signin = () => {
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+
+  function togglePasswordVisibility() {
+    setIsPasswordVisible((prevState: any) => !prevState);
+  }
+  const router = useRouter();
+  const email = useRef('');
+  const password = useRef('');
+  const handleSubmit = async (e: React.FormEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    const res = await signIn('credentials', {
+      email: email.current,
+      password: password.current,
+      redirect: false,
+    });
+
+    if (!res?.error) {
```

Comment:
I think it would be also better to add an alert or something to notify the user about the error.

Reply:
I'll do it now

Reply:
Added Sonner toast..

![image](https://github.com/code100x/cms/assets/72003665/69314d57-0c89-49c3-82af-7621cf97fd7c)